.github/workflows/ci.yaml

```diff
@@ -146,4 +146,4 @@ jobs:
           argocd app wait -l "argocd.argoproj.io/instance=science-platform" \
             --port-forward \
             --port-forward-namespace argocd \
-            --timeout 300
+            --timeout 600
```

Comment:
My experience is that if it doesn't come up after five minutes, it's never going to come up.

Reply:
Yeah, that was my thinking too until it turned right up on my laptop.  I'm going to try a few other things but this one will be reverted.  Just wanted to make sure it wasn't that